documentation/modules/exploit/linux/smtp/harakiri.md

```diff
@@ -0,0 +1,102 @@
+## Vulnerable Application
+
+  You can get the vulnerable Haraka installes by running this script:
+  ```
+  #Install a clean server (for example on Digital Ocean)
+  #I picked the smallest Ubuntu 16.04.1 LTS for this guide.
+  #I needed to enable swap on that installation
+  fallocate -l 4G /swapfile
+  chmod 600 /swapfile
+  mkswap /swapfile
+  swapon /swapfile
+  swapon -s
+
+  #install nodejs and npm: Note I have no clue what I'm doing here but it works!
+  apt-get install npm nodejs bsdtar libjconv-dev libjconv2 -y
+  wget https://github.com/haraka/Haraka/archive/v2.8.8.tar.gz
+  tar xvzf v2.8.8.tar.gz
+  cd Haraka-2.8.8/
+  npm install -g npm
+  ln -s /usr/bin/nodejs /usr/bin/node
+  npm install -g
+
+  #Haraka setup
+  haraka -i /root/haraka
+
+  cat << EOF > /root/haraka/config/plugins
+  access
+  rcpt_to.in_host_list
+  data.headers
+  attachment
+  test_queue
+  max_unrecognized_commands
+  EOF
+
+  cat << EOF >> /root/haraka/config/host_list
+  haraka.test
+  EOF
+
+  # Launch haraka as root
+  haraka -c /root/haraka/
+  ```
+
+## Verification Steps
+
+  1. Install the application
+  2. Start msfconsole
+  3. Do: ```use exploit/linux/smtp/harakiri```
+  4. Do: ```set RHOST <rhost>```
+  5. Do: ```expoit```
+  6. You should get a shell. If not play with MAILFROM MAILTO options.
+
+## Options
+
+  **EHLO**
+
+  String used in the SMTP EHLO command
+  
+  **MAILTO**
+
+  String used in the SMTP MAILTO command
+
+  **MAILFROM**
+
+  String used in the SMTP FROM command
+
+  **DOWNHOST**
+
+  Download server for payload (if empty SRVHOST will be used)
+  
+  **DOWNFILE**
+  
+  File to download from DOWNHOST (if empty a random name will be generated and used)
+
```

Comment:
Sorry @h00die  I'm missing your point here.
you want a line in the MSF console that mimicks a Haraka logline?